src/lib/utils/md.ts

```diff
@@ -91,9 +92,9 @@ export const getContentBySlug = (slug: string) => {
   const realSlug = `${slug}/index.md`
   const fullPath = join(CURRENT_CONTENT_DIR, realSlug)
   const fileContents = fs.readFileSync(fullPath, "utf8")
-  const { data: frontmatter, content } = matter(fileContents)
-
-  const items: PageContent = {
+  const { data, content } = matter(fileContents)
+  const frontmatter = data as Frontmatter
```

Comment:
Typing as union type `Frontmatter` at this point, which will allow front matter formatted for _any_ of the acceptable layouts.